recipe/recipe.yaml

```
@@ -155,6 +155,8 @@ outputs:
         - xxhash
         - zlib
         - zstd
+        # Required for roofit_multiprocess
+        - cppzmq
```

Given https://github.com/conda-forge/root-feedstock/pull/292#issuecomment-2858807856 should verify that `libzmq` is not in the global pinned feedstock, and if not, then add bounds to `zeromq`.

```suggestion
        # Required for roofit_multiprocess
        - cppzmq
        # Required zmq_ppoll API added in v4.3.5
        - zeromq >= 4.3.5
```